menu/menu.yaml

```diff
@@ -118,6 +118,26 @@
       - Title: Observability
         Url: architecture/aws/observability
 
+- Name: Particular Service Platform
+  Topics:
+  - Title: General
+    Articles: 
+    - Url: platform
+      Title: About the Service Platform
```

Comment:
On second thought, maybe `About the platform` is less awkward. It's under a menu that has the full platform name so it should be obvious what platform it refers to